### PR TITLE
Add the FractalZoo repository

### PR DESCRIPTION
**Pull request by comodoro**

### CODE REVIEW

File: projects/comodoro/FractalZoo.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tags:
+  - Android
+avatar: fractalzoo.png
+featuredMap: # (optional) A CodeSee Map to serve as a visual starting point for contributors
+  url: https://app.codesee.io/maps/public/a0d9c5e0-24e5-11ec-992f-b1502cc33c49
+  description: Layout is that of a standard Java Android app
+  url: https://app.codesee.io/maps/public/3c316a10-24d8-11ec-a724-79d7dd589517
+  description: Goal of this repository is efficient rendering with shaders
+  url: https://app.codesee.io/maps/public/d894bea0-24da-11ec-93f3-dbc55dad4855
+  description: Rendering code is separate for standard Canvas and OpenGL
```
Comment (deammer) [inline projects/comodoro/FractalZoo.mdx:17]: Hi @comodoro, thank you for your PR! The `featuredMap` field doesn't currently support multiple maps. I would suggest 2 approaches:
1. [add custom labels](https://docs.codesee.io/en/latest/guide/#labels-and-colors) to denote the various parts of your codebase, or
2. keep a single `featuredMap` and include links to the other 2 in your mdx file

Comment (comodoro) [reply projects/comodoro/FractalZoo.mdx]: Sorry, I did not notice it in the preview. I have added some nice colors:) By the way, the tool icons could use some alt text/tooltips.

Comment (deammer) [reply projects/comodoro/FractalZoo.mdx]: I agree, and I'll pass that on to the team for prioritization. Thank you!

**Review by deammer — APPROVED** [review]
Thank you @comodoro! This will go live in a few minutes.

**Pull request merged**